libraries/python/skills/skills/document-skill/document_skill/__init__.py

```diff
@@ -1,9 +1,9 @@
-from document_skill.document_skill import Attachment, DocumentContext  # temp for dev testing purposes
+from document_skill.document_skill import Attachment, DocumentSkillContext
 from document_skill.document_skill import DocumentSkill
 
 
 __all__ = [
     "Attachment",
-    "DocumentContext",
+    "DocumentSkillContext",
```

Comment:
FYI, this skill is being used in Prospector as well now, so changes like this will need to be updated there too.